etc/i2prouter.profile

```diff
@@ -67,5 +67,5 @@ shell none
 disable-mnt
 private-cache
 private-dev
-private-etc alternatives,ca-certificates,crypto-policies,i2p,java-8-openjdk,pki,ssl
+private-etc alternatives,ca-certificates,crypto-policies,dconf,group,hostname,hosts,i2p,java-8-openjdk,java-9-openjdk,java-10-openjdk,java-11-openjdk,java-12-openjdk,java-13-openjdk,java-openjdk,ld.so.cache,localtime,machine-id,nsswitch.conf,passwd,pki,resolv.conf,ssl
```

Comment:
Perhaps we can get away with `java-*-openjdk` instead?

Reply:
I tried that at first, but I don't know if private-etc accepts regular expressions. I keep getting:
```
Error: "java-*-openjdk" is an invalid filename
Error: proc 10805 cannot sync with peer: unexpected EOF
Peer 10806 unexpectedly exited with status 1
```
So I just decided to manually enter the versions.